src/components/FeatureWalkthrough.tsx

```diff
@@ -0,0 +1,128 @@
+import { FunctionComponent, useEffect } from 'react'
+
+import classNames from 'classnames'
+import { startCase } from 'lodash'
+import Link from 'next/link'
+
+import { ContentSection } from '@components'
+import { buttonStyle, buttonLocation } from '@data'
+import { Features, VideoElement } from '@interfaces/featureWalkthrough'
+
+import { COLORS } from './ContentSection'
+
+interface Props {
+    features: Features[]
+    className?: string
+    color?: keyof typeof COLORS
+}
+
+export const FeatureWalkthrough: FunctionComponent<Props> = ({ features, className = '', color = 'none' }) => {
```

Comment:
I'd vote to nix the `className` prop to eliminate drilling CSS props, and wrap `FeatureWalkthrough` in `<ContentSection>` so padding or bg's can be added there. What do you think?

Reply:
This solution is much easier to maintain. I think that it also makes sense to remove the `color` prop and the `ContentSection` within this component. By removing `ContentSection`, we prevent double wrapping the component. How does this sound to you? See: d31cbf5